superset-frontend/src/cccs-viz/plugins/plugin-chart-ag-grid/src/maps.ts

```diff
@@ -0,0 +1,29 @@
+import CountryValueRenderer from './renderers/CountryValueRenderer';
+import DomainValueRenderer from './renderers/DomainValueRenderer';
+import Ipv6ValueRenderer from './renderers/Ipv6ValueRenderer';
+import JsonValueRenderer from './renderers/JsonValueRenderer';
+import TimestampValueRenderer from './renderers/TimestampValueRenderer';
+
+// Key is column advanced type, value is renderer
+export const rendererMap = {
+  IPV6: Ipv6ValueRenderer,
+  DOMAIN: DomainValueRenderer,
+  COUNTRY: CountryValueRenderer,
+  JSON: JsonValueRenderer,
+  DATE: TimestampValueRenderer,
+  DATETIME: TimestampValueRenderer,
+  'TIMESTAMP WITHOUT TIME ZONE': TimestampValueRenderer,
+  'TIMESTAMP WITH TIME ZONE': TimestampValueRenderer,
+};
+
+export const formatIpV4 = (v: any) => {
```

Comment:
This capital V in IPv4 is everywhere now and it's killing me inside lol! Ipv4!!! (I'm just complaining, not insisting it change.)